site/docs/v1/tech/apis/_key-generate-post-request-body.adoc

```diff
@@ -14,6 +14,9 @@ The algorithm used to generate the Key. The following values represent algorithm
 * `HS384` - HMAC using SHA-384 hash algorithm
 * `HS512` - HMAC using SHA-512 hash algorithm
 
+[field]#key.issuer# [type]#[String]# [optional]#Optional#::
+The issuer of the RSA or EC certificate. If omitted,this value will default to the issuer defined on the tenant. For HMAC keys, this field does not apply and will be ignored if specified.
```

Comment:
Since this is an API doc, it is helpful to tell the developer the exact value or field you are referencing. Also, it isn't just any tenant, this API doesn't know anything about tenants, so we need to call out which tenant we will use. So I would say it something like this:

> If omitted, this value will default to the value of [field]#tenant.issuer# on the default tenant.